deploy/helm/sumologic/templates/setup/job.yaml

```
@@ -28,6 +29,7 @@ spec:
 {{- end }}
       labels:
 {{- if .Values.sumologic.podLabels }}
+{{- include "sumologic.labels.otelcol.events" . | nindent 8 }}
```

```suggestion
{{- include "sumologic.labels.otelcol.events" . | nindent 8 }}
{{- if .Values.sumologic.podLabels }}
```
this include should be outside of `if` and also `events` shouldn't be in `setup`

---

fixed